specification/components/schemas/extensions/Communication.yaml

```diff
@@ -12,7 +12,10 @@ properties:
       - https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-NHSCommunication
   extension:
     type: array
-    description: Definition of communication extension, The array must have two items, a valueCodeableConcept for the language and a valueBoolean for if an interpreter is required.
+    description: Definition of communication extension.
```

Comment:
I had to go searching for why you had two empty lines in here

It may have been better to use one of the other 8 😱 types of YAML multiline string for this. But 🤷‍♂️

https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines